itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestExportImport.java

```diff
@@ -106,6 +113,54 @@ public void dataImportAfterMetadataOnlyImport() throws Throwable {
         .verifyResults(new String[] { "1", "2" });
   }
 
+  @Test
+  public void testExportExternalTableSetFalse() throws Throwable {
+    String path = "hdfs:///tmp/" + dbName + "/";
+    String exportMDPath = "'" + path + "1/'";
+    String exportDataPath = "'" + path + "2/'";
+    String exportDataPathRepl = "'" + path + "3/'";
+    srcHiveWarehouse.run("create external table " + dbName + ".t1 (i int)")
+            .run("insert into table " + dbName + ".t1 values (1),(2)")
+            .run("export table " + dbName + ".t1 to " + exportMDPath + " for metadata replication('1')")
```

Comment:
Whether metadata-only dump is equivalent to setting hive.repl.dump.metadata.only=true?